styles.css

```diff
@@ -0,0 +1,143 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+.main-container {
+  padding: 0 3%;
+}
+
+.nav_container {
+  max-width: 80rem;
+  margin: 0 auto;
+  padding: 0 3%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 0.5rem;
+}
+
+.nav_logo {
+  padding: 10px 12px;
+  font-size: 18px;
+  color: #162ae3;
+  display: flex;
+}
+
+.nav_head {
+  text-decoration: none;
+  color: #162ae3;
+}
+
+.nav_link {
+  padding: 10px 12px;
+  margin: 0 0.5rem 0 0;
+  color: #344563;
+  font-weight: 500;
+  font-size: 1rem;
+  text-decoration: none;
+}
+
+.toolbar {
+  display: flex;
+}
+
+.footer_head {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  color: #7f8cff;
+  text-transform: uppercase;
+  margin-top: -10px;
+}
+
+.social-icon {
+  margin-top: -5px;
+}
+
+.social-icon ul {
+  display: flex;
+  margin: 0 1rem 0 0;
+  padding: 0;
+  gap: 20px;
+}
+
+.social-icon ul li {
+  list-style: none;
+}
+
+.section {
+  padding: 0 3%;
+  background-image: url(Header-bg.png);
```

Comment:
- [ ] You should use the background image for mobile version instead. You can find it in the Figma design under `Template-1 Mobile Main` with a name of `header-shapes mobile`. You may export that svg image and use that image for mobile version.

Reply:
thanks I have added the header-shapes mobile image the svg format for mobile version